src/utils/instance-resolver.ts

```diff
@@ -0,0 +1,11 @@
+import {API, hash} from 'mesg-js/lib/api/types'
+import {resolveSID} from 'mesg-js/lib/util/resolve'
+
+export default async (api: API, sidOrHash: hash | string): Promise<hash> => {
```

Comment:
This can resolve instance hash and sid but not service hash?

Why not doing a function that resolve both service hash and sid to instanceHash?

Reply:
same for the service resolver. This can be added, I will add an issue on that but this can be done later on. The goal is really to have something simple for the user and they will most likely use the sid